include/conversation.php

```diff
@@ -33,10 +33,11 @@
 use Friendica\DI;
 use Friendica\Model\Contact;
 use Friendica\Model\Item;
+use Friendica\Model\Post;
 use Friendica\Model\Profile;
 use Friendica\Model\Tag;
 use Friendica\Model\Verb;
-use Friendica\Object\Post;
+use Friendica\Object\Post as ObjectPost;
```

Comment:
More natural in English.
```suggestion
use Friendica\Object\Post as PostObject;
```

Reply:
That had been done automatically by my IDE :-)

Reply:
It makes sense with the order of the namespace but not in natural English.